### PR TITLE
Export csv

### PR DESCRIPTION
**Pull request by bkambo**
Implemented Export CSV. Can Import by: Request ID, date range, and all requests

### CODE REVIEW

File: scripts/js/exportCSV.jsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                        for (var i=0; i < jsonResponse.length; i++) {
+                            var tempRequest = {};
+                            var id = jsonResponse[i].id;
+                            tempRequest.id = id;
+                            RequestIDs.push(tempRequest)
+                        }
+                        this.setState({
+                            requestIds: RequestIDs
+                        });
+                        console.log(this.state.requestIds);
```
Comment (alekhrycaiko) [inline scripts/js/exportCSV.jsx]: can we remove the logging

**Comment by alekhrycaiko** [comment]
react-bootstrap-date-picker needs to be added to package.json

File: scripts/js/exportCSV.jsx [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                if (err) {
+                    this.setState({
+                        statusMessage: err,
+                        statusDisplay: true,
+                        errorMessage: "error"
+                    });
+                } else {
+                    try {
+                        var d = new Date()
+                        var fullDate = d.getFullYear() + (("0" + d.getMonth()).slice(-2)) + ("0" + d.getDate()).slice(-2)
```
Comment (alekhrycaiko) [inline scripts/js/exportCSV.jsx]: What're we trying to do here?
Just a heads up that you can use momentjs to deal with any date formatting if that makes things simpler.

Comment (bkambo) [inline scripts/js/exportCSV.jsx]: this is to create a file name for the file that is going to be downloaded - was just easier to format it this way according to what Bader specified

**Pull request merged**